src/actions/objects.js

```diff
@@ -235,7 +235,9 @@ export const getAllObjects = (fromIndex=0) => {
   }
 };
 
-export const getRelatedObjects = (objectID, fromIndex=0) => {
+export const getRelatedObjects = (value, objectID, fromIndex=0) => {
```

Comment:
This is fine, but do you want to shift value to the second param and give it a default of 50 so you don't need to always include it?

Reply:
Good idea.